### PR TITLE
Adding replicas.py

### PR DESCRIPTION
**Pull request by PiotrProkop**

### CODE REVIEW

**Comment by PiotrProkop** [comment]
@ss7pro @DTadrzak please review

File: mariadb/manifest.jsonnet [hunk 1]
```diff
@@ -126,7 +126,12 @@ kpm.package({
       name: "mariadb-tuning",
       type: "configmap",
     },
-
+    {
```
Comment (DTadrzak) [inline mariadb/manifest.jsonnet]: add new line

File: mariadb/manifest.jsonnet [hunk 2]
```diff
@@ -126,7 +126,12 @@ kpm.package({
...
+      file: "configmaps/replicas.py.yaml.j2",
+      template: (importstr "templates/configmaps/replicas.py.yaml.j2"),
+      name: "mariadb-replicas",
+      type: "configmap",
+    },
```
Comment (DTadrzak) [inline mariadb/manifest.jsonnet]: add new line

File: mariadb/templates/configmaps/replicas.py.yaml.j2 [hunk 3]
```diff
@@ -0,0 +1,53 @@
+apiVersion: v1
+kind: ConfigMap
+data:
+  replicas.py: |+
+    #!/usr/bin/env python
+    import sys
```
Comment (DTadrzak) [inline mariadb/templates/configmaps/replicas.py.yaml.j2]: use alphabetical order

Comment (DTadrzak) [inline mariadb/templates/configmaps/replicas.py.yaml.j2]: delete unused imports

File: mariadb/templates/configmaps/replicas.py.yaml.j2 [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+data:
+  replicas.py: |+
+    #!/usr/bin/env python
+    import json
+    import os
+    import ssl
+    import sys
+    import urllib2
+
+    URL = ('https://kubernetes.default.svc.cluster.local/apis/extensions/v1beta1/daemonsets')
```
Comment (ss7pro) [inline mariadb/templates/configmaps/replicas.py.yaml.j2]: cluster.local should be replaced with {{ network.kubernetes_domain }}

File: mariadb/templates/configmaps/replicas.py.yaml.j2 [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+       # parse to dict
+       json_acceptable_string = data.replace("'", "\"")
+       output = json.loads(json_acceptable_string)
+
+       return output
+
+    def main():
+       reply = get_daemonsets()
+       name = "mariadb"
+       namespace = os.environ["NAMESPACE"]
```
Comment (ss7pro) [inline mariadb/templates/configmaps/replicas.py.yaml.j2]: if not namespace:
  namespace = 'default'

**Comment by ss7pro** [comment]
LGTM

**Pull request merged**